x/pairing/types/relay_exchange.go

```diff
@@ -0,0 +1,57 @@
+package types
+
+import (
+	"bytes"
+	"encoding/binary"
+	"strings"
+
+	"github.com/lavanet/lava/utils"
+)
+
+type RelayExchange struct {
+	Request RelayRequest
+	Reply   RelayReply
+}
+
+func NewRelayExchange(req RelayRequest, res RelayReply) RelayExchange {
+	return RelayExchange{Request: req, Reply: res}
+}
+
+func (re RelayExchange) GetSignature() []byte {
+	return re.Reply.Sig
+}
+
+func (re RelayExchange) PrepareForSignature() []byte {
+	re.Reply.Sig = []byte{}
+	metadataBytes := make([]byte, 0)
```

Comment:
make() is redundant here (unless you can forecast the final size and then pre-allocate as much). var metadataBytes []byte suffices.

Reply:
done. see https://github.com/lavanet/lava/commit/f2e3955a33de4da6d20894a8334eae0acb89e2dd